OrbitCore/Utils.h

```diff
@@ -275,6 +275,35 @@ class CWindowsMessageToString {
 };
 #endif
 
+//-----------------------------------------------------------------------------
+inline std::string ShortenStringWithEllipsis(std::string_view text,
+                                             int max_len) {
+  if (max_len < 0) {
+    return std::string(text);
+  }
+
+  size_t max_len_unsigned = static_cast<size_t>(max_len);
+  if (max_len_unsigned <= 3) {
```

Comment:
Maybe define 3 as a constant. The name can indicate how it is used:

constexpr const size_t kNumCharsEllipsis = 3;